### PR TITLE
Fix Fortify small typos

### PR DESCRIPTION
**Pull request by teoberi**
Cosmetic changes for Fortify

### CODE REVIEW

File: src/functions/proccheck.sh [hunk 1]
```diff
@@ -117,7 +117,7 @@ proccheck() {
     fi
   fi
 
-  #check for forifty source support
```
Comment (slimm609) [inline src/functions/proccheck.sh:120]: make build needs run before checking in.   

the `hack/enable-git-hooks.sh` script will enable the pre-commit hooks which will prevent committing without the render and do it for you so then its just adding it.

**Comment by slimm609** [comment]
https://github.com/slimm609/checksec.sh/actions/runs/8858421903/job/24327044367?pr=239

**Comment by slimm609** [comment]
tests are failing because you need to run `make build` and check in the results

**Comment by teoberi** [comment]
> tests are failing because you need to run `make build` and check in the results

I don't use git locally.

**Comment by slimm609** [comment]
> > tests are failing because you need to run `make build` and check in the results
> 
> I don't use git locally.

I will approve this one and generate it but for future MRs, I will not accept them without following the process so that the tests can properly run.

**Comment by teoberi** [comment]
OK. I understood, I will take the necessary steps. Thank you.

**Pull request merged**